client/e2e/poms/dashboard.ts

```diff
@@ -36,68 +42,88 @@ export class DashboardPOM {
 
   constructor(page: Page) {
     this.page = page;
-    this.msgPending = this.page.locator(DataTestID.MESSAGE_PENDING);
-    this.selectOperatorTile = this.page.getByText(
-      "1 pending action(s) required",
+    this.messagePending = page.getByTestId(DataTestID.MESSAGE_PENDING);
+    this.operationsTile = page.getByRole("link", {
+      name: TileTextDashboard.TILE_OPERATIONS,
+    });
+    this.operationsTileIndustry = page.getByRole("link", {
+      name: TileTextDashboard.TILE_OPERATIONS,
+    });
+    this.operatorsTile = page.getByRole("link", {
+      name: TileTextDashboard.TILE_OPERATORS,
+    });
+    this.operatorsTileIndustry = page.getByText(
+      TileTextDashboard.TILE_OPERATOR_MINE,
     );
-
-    this.page = page;
-    this.operationsTile = page.getByRole("link", { name: /.*operations.*/i });
-    this.operatorsTile = page.getByRole("link", { name: /.*operator.*/i });
-    // the id to select these as these as we will start adding notifications to the tiles
-    // the operators tile link text is just `0 pending action(s) required`
-    this.operatorsTileIndustry = page.locator("#My-Operator-link");
-    this.operationsTileIndustry = page.locator("#My-Operations-link");
     this.reportProblemLink = page.getByRole("link", {
-      name: "Report problems to GHGRegulator@gov.bc.ca",
+      name: TileTextDashboard.TILE_REPORT_PROBLEM,
     });
-    this.userAccessManagementTileIndustry = page.locator(
-      "#User-Access-Management-link",
+    this.selectOperatorTile = page.getByText(
+      TileTextDashboard.TILE_OPERATOR_SELECT,
```

Comment:
I don't feel too strongly about this but I wonder if we could somehow make a data-testid for each tile type that used this format and would be user independent eg: `tile-operator`, `tile-operation`, `tile-user` etc. That way we wouldn't have to worry about users or label changes.

Reply:
@marcellmueller great discussion point for the team to weigh-in on: if the role or text value is important then consider using user facing locators OR if test resiliency is prioritized the consider using test ids.